docs/reference/workflow/metrics.md

```diff
@@ -1,3 +0,0 @@
-# `kolena.workflow.metrics`
```

Comment:
i think we should still keep this doc for backward compatibility

Reply:
This results in breaking relative links in metrics docstrings. Will require additional changes to support